[FR] Update Beatmap Editor/Timing (#3059)
ppy/osu-wiki: wiki/Beatmap_Editor/Timing/fr.md

# Onglet Timing

*Voir également : [Comment timer les chansons](/wiki/How_to_time_songs)*

> ![Menu Timing ](img/Timing_base.jpg "Menu Timing")

Reviewer: ```suggestion
![Menu Timing](img/Timing_base.jpg "Menu Timing")
```

Le timing d'un [beatmap](/wiki/Beatmaps) consiste en un décalage de la chanson et un BPM (battements par minute). Le décalage de la chanson est la quantité de temps entre le début d'un fichier audio et le premier battement de la chanson. Le BPM de la chanson est essentiellement la vitesse de la chanson.

Un timing correct est impératif dans le processus de beatmapping pour permettre de placer les objets à temps avec le rythme.

## Setup

> ![Screenshot of the "Timing and Control Points" window](img/TimingSetup.jpg "Le dialogue de configuration du timing montre les fluctuations du BPM de la carte de rythme et les changements de jeu d'échantillons")

Reviewer 2: ```suggestion
![Screenshot of the "Timing and Control Points" window](img/TimingSetup.jpg "Le dialogue de configuration du timing montre les fluctuations du BPM de la map et les changements de jeu d'échantillons")
```

> ![Capture d'écran de la fenêtre "Chronométrage et points de contrôle](img/TimingSetup.jpg "Le dialogue de configuration du timing montre les fluctuations du BPM de la map et les changements de jeu d'échantillons")

Reviewer: ```
![Capture d'écran de la fenêtre "Chronométrage et[...]
                                ^
```
This quotation mark is missing it's other quotation mark.

Les sections de chronométrage vous permettent de varier le chronométrage dans la chanson. Il existe deux types de sections de chronométrage : une section de chronométrage régulier et une section de chronométrage hérité.

> Une section de minutage régulière est affichée sous forme de ligne rouge dans l'éditeur. Chaque beatmap doit en avoir au moins une. Cependant, certains beatmaps changent leur BPM pendant le morceau et nécessitent par conséquent plusieurs sections de timing régulier.

Reviewer 2: ```suggestion
Une section de minutage régulière est affichée sous forme de ligne rouge dans l'éditeur. Chaque beatmap doit en avoir au moins une. Cependant, certains beatmaps changent de BPM pendant le morceau et nécessitent par conséquent plusieurs sections de timing régulier.
```

> Les sections de chronométrage héritées sont affichées sous forme de ligne verte dans l'éditeur. Elles ne permettent pas de modifier le timing de la carte des temps, mais se concentrent sur l'ajustement du timing du morceau. Les sections de timing héritées vous permettent de modifier le timing de la carte des temps et le volume des [hitsound's](/wiki/hit_sound), les frappeurs eux-mêmes, et [slider velocity](/wiki/slider_velocity), etc. pour cette section sans modifier le timing du morceau (ce qui se produirait avec une section de timing normale). Ces deux types de sections de timing sont aussi communément appelés respectivement _décalages rouges_ et _décalages verts_.

Reviewer 2: ```suggestion
Les sections de chronométrage héritées sont affichées sous forme de ligne verte dans l'éditeur. Elles ne permettent pas de modifier le timing de la carte des temps, mais se concentrent sur l'ajustement du timing du morceau. Les sections de timing héritées vous permettent de modifier le timing de la carte des temps et le volume des [hitsound's](/wiki/hit_sound), les frappeurs eux-mêmes, et la [vitesse des sliders](/wiki/slider_velocity), etc. pour cette section sans modifier le timing du morceau (ce qui se produirait avec une section de timing normale). Ces deux types de sections de timing sont aussi communément appelés respectivement _décalages rouges_ et _décalages verts_.
```
